### ISSUE TITLE
[SPIKE] Reassess minScore for accessibility checks

### ISSUE THREAD

**Issue by lexjoseph** [PublicDataWorks/police_data_manager#122]
**Describe what needs to be investigated**
Our current minScore for accessibility checks is 70. We want to know if this should be higher. 

**To Reproduce**
1. Go to the lighthouse job in pipeline
2. Expand the "lhci autorun" tab
3. View each report to see accessibility scores 

**Expected behavior**
1. Determine whether or not we should bump up the minScore based on the accessibility score findings. 
2. If so, change minScore value for accessibility in `lighthouserc.json`. If not, do nothing.
